wrench/src/reftest.rs

```diff
@@ -96,6 +109,65 @@ impl ReftestImage {
     }
 }
 
+enum ReftestScene {
+    Blank,
```

Comment:
I don't think `Blank` logic here is worth it, given that it could be replaced by a single `wrench/reftests/gradient/blank.yaml` file